fem/docker-compose-fem.yml

```diff
@@ -77,45 +77,44 @@ services:
     container_name: fem_opensearch_exporter
     command:
      - '--es.uri=http://fem_opensearch:9200'
-    restart: unless-stopped
     networks:
       - fem_monitoring
     depends_on:
       - fem_opensearch
   fem_jaeger-es-cleaner:
-    image: jaegertracing/jaeger-es-index-cleaner:1.39
+    image: jaegertracing/jaeger-es-index-cleaner:1.42
     container_name: fem_opensearch-cleaner
     command: [
       "7",
       "http://fem_opensearch:9200"
     ]
     networks:
-    - fem_monitoring
+      - fem_monitoring
```

Comment:
@ronnyfriedland , I think the definition of the project name is needed now, e.g. add: "--project-name fem".
With the separation into modules it must be ensure that all components run in the same network.
As you know without configure the project name, the name for the container as well as the name for the network were created from the directory name by default. That results into different subnets for the containers. The avoid that we have to set the project name and remove the "fem_"-prefix in all docker-compose files.